introduction.md

```diff
@@ -1,3 +1,8 @@
 Hi! My name's Melissa.
 
-My favorite CodeSchool course track was the JavaScript track...but I'm liking the Git Tutorial too!
\ No newline at end of file
+My favorite CodeSchool course track was the JavaScript track...but I'm liking the Git Tutorial too!
+
+*Git
+*JavaScript
+*Ruby on Rails
+*Apache Framework
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?